resources/web/template.html

```diff
@@ -161,1115 +101,314 @@
+          if (!event.target.matches('.dropdown-btn')) {
+    
+            var dropdowns = $(".dropdown-content");
+            var i;
+            for (i = 0; i < dropdowns.length; i++) {
+              var openDropdown = dropdowns[i];
+              if (openDropdown.classList.contains('show')) {
+                openDropdown.classList.remove('show');
+              }
+            }
           }
-        }
-      }
-    });
-
-    // if($(window).width() < 769){
-    //   var $item = $('.scrollable-panel li'), 
-    //       visible = 3, //Set the number of items that will be visible
-    //       index = 0, //Starting index
-    //       endIndex = ( $item.length / visible ) - 1; //End index (NOTE:: Requires visible to be a factor of $item.length... You can improve this by rounding...)
-
-    //   $('#right-arrow').click(function(){
-    //     if(index < endIndex ){
-    //       index++;
-    //       $item.animate({'left':'-=300px'});
-    //     }
-    //   });
-
-    //   $('#left-arrow').click(function(){
-    //     if(index > 0){
-    //       index--;            
-    //       $item.animate({'left':'+=300px'});
-    //     }
-    //   });
-    // }
-  });
-  $(window).resize(function () {
-    $('.horizon-swiper').horizonSwiper({
-      showItems: 3,
-      arrows: true
-    });
-  });
-</script>
+        });
     
-      
-
-
-  
-
-
-
-
-
-  
-  
-  
-  
-    <div class="tertiary-nav bdr-btm-e0e0e0">
-      <div class="container">
-        <div class="p-t-b-15 clearfix">
-          <div class="breadcrum-wrapper pull-left text-left">
-          
-            <a href="/guide">Docs</a> 
-          
-          </div>
-          
-          
-          
-        </div>
-      </div>
-    </div>
-
-    <div class="nav-mobile-dropdown hidden-sm hidden-md hidden-lg bdr-btm-e0e0e0 clearfix">
-      <div class="container text-right">
-        <div class="breadcrum-wrapper pull-left text-left">
-          
-              <a href="/guide">Docs</a>
-          
-        </div>
-        
-        
-      </div>
-    </div>
-  
-
-
-<style type="text/css">
-
-  
-</style>
+        // if($(window).width() < 769){
+        //   var $item = $('.scrollable-panel li'), 
+        //       visible = 3, //Set the number of items that will be visible
+        //       index = 0, //Starting index
+        //       endIndex = ( $item.length / visible ) - 1; //End index (NOTE:: Requires visible to be a factor of $item.length... You can improve this by rounding...)
     
+        //   $('#right-arrow').click(function(){
+        //     if(index < endIndex ){
+        //       index++;
+        //       $item.animate({'left':'-=300px'});
+        //     }
+        //   });
+    
+        //   $('#left-arrow').click(function(){
+        //     if(index > 0){
+        //       index--;            
+        //       $item.animate({'left':'+=300px'});
+        //     }
+        //   });
+        // }
+      });
+      $(window).resize(function () {
```

Comment:
Should we move this into docs.js?

Reply:
Ah. I see. It was in there before. We *probably* should move it to docs.js but it can totally wait in that case.

Reply:
++ I'll plan on moving this later this week.